src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -249,11 +255,20 @@ public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceNa
             }
             catch (HttpRequestException)
             {
-                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider",
-                    $"message:Observer SQL query request failed, query:{query}, statusCode:{response.StatusCode}, response:{result}");
+                var logMessage = $"message:Observer SQL query request failed, route:{route}, query:{query}, " +
+                    $"statusCode:{response.StatusCode}, response:{result}, latencyMs:{timer.ElapsedMilliseconds}";
```

Comment:
We already log latency in event id `3002` which is called in `DataLogProviderDecoreator` See `MakeDependencyCall` in `DataLogProviderDecorator` for more details.